tests/Tests/500__jupyterhub/minimal-pytorch-test.robot

```diff
@@ -40,7 +40,7 @@ Verify PyTorch Image Can Be Spawned With GPU
     ...     ODS-1145
     Clean Up Server
     Stop JupyterLab Notebook Server
-    Handle Start My Server
+    Fix Spawner Status
```

Comment:
maybe lines 41-41-44 are not needed anymore since `Fix Spawner Status` is used, are they?

Reply:
Theoretically no, but to err on the side of caution I'd prefer to explicitly close the previously running server and then fixing any potential issues with `Fix Spawner Status` since we have seen some failures here

Reply:
hm I don't like it much, but if you think it is necessary now to make the suite more robust it's okay to keep it.
However, I think we should revisit the Fix Spawner Status and try to make it more reliable to avoid a such duplication of code

Reply:
Yeah, agreed that `Fix Spawner Status` probably needs to be revisited to make it more robust. For now I'd prefer to keep it like this in order to fix during this test cycle.